extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -325,15 +326,19 @@ describes.realWin(
 
       it('should expand the promptUISrc', async () => {
         const config = dict({
-          'promptUISrc': 'https://example.test/?cid=CLIENT_ID&r=RANDOM',
+          'promptUISrc':
+            'https://example.test/?cid=CLIENT_ID&r=RANDOM&clientconfig=CONSENT_INFO(clientConfig)&cpid=CONSENT_PAGE_VIEW_ID_64',
           'clientConfig': {
             'test': 'ABC',
           },
         });
         consentUI = new ConsentUI(mockInstance, config);
         consentUI.show(false);
         await macroTask();
-        expect(consentUI.ui_.src).to.match(/cid=amp-.{22}&r=RANDOM/);
+        await macroTask();
+        expect(consentUI.ui_.src).to.match(
+          /cid=amp-.{22}&r=RANDOM&clientconfig=.{28}&cpid=consent_page_view_id_64/
```

Comment:
1. `foo_consent_page_view_id_64_123`, split, and `RANDOM` at the end like above like above.
2. `.{28}` is not a good pattern to check the `CONSENT_INFO` result. Let's expect the actual value.

```suggestion
          new RegExp(
            'cid=amp-.{22}&' +
              `clientconfig=${encodeURIComponent(
                JSON.stringify(config.clientConfig)
              )}&` +
              'cpid=consent_page_view_id_64&' +
              'r=RANDOM'
          )
```